contexts/chat.tsx

```diff
@@ -336,7 +334,7 @@ const ChatContextProvider: React.FC<ChatContextProps> = ({
       setScript(script);
       setThread(newThread.meta.id);
       setSelectedThreadId(newThread.meta.id);
-      setWorkspaceDir(newThread.meta.workspace);
+      setWorkspaceDir(newThread.meta.workspace).then();
```

Comment:
Hmm what is this?

Reply:
Will remove. This was just a to get rid of a squiggly line that was bothering me.

Reply:
Done.